### ISSUE TITLE
Broken vhost template rendering in latest release

### ISSUE THREAD

**Issue by neclimdul** [lando/nginx#39]
From slack I was pointed to changes in the render template tooling. I was actually relying on that to build custom vhost for our environment and that's now broken.

Specifically, each vhost contains a line like this using the LANDO_APP_COMMON_NAME environment variable to resolve the correct common name at compile time. 

```
server_name "backstop.{{LANDO_APP_COMMON_NAME}}.lndo.site";
```

This is useful because I have a clean release management version of the project and a often broken development version of the project with different lando names in a `.lando.local.yml` file and this correctly handled resolving the domain.

**Comment by neclimdul** [comment]
I see the problem now and I think its a road I've been down before because I've got a solution of sorts in one of my nginx containers. 

In the containers I was using it in I was able to use `envsubst` as a quick stand in. It does a very similar thing as a more unixy tool. But despite this being a standard gettext cli tool, a quick look at the the bitnami container shows it is not available. I assume could lead to some whack-a-mole guessing at what variables users need and additional sed lines. :(
